SPECS/mstflint/mstflint.spec

```diff
@@ -29,35 +30,114 @@ Requires:       python3
 This package contains firmware update tool, vpd dump and register dump tools
 for network adapters based on Mellanox Technologies chips.
 
-%prep
-%autosetup -p1
+%package devel
+Summary:        Scripts and header files for mstflint
+Requires:       %{name} = %{version}-1
+Provides:       mstflint-devel = %{version}-1
 
-find . -type f -iname '*.[ch]' -exec chmod a-x '{}' ';'
-find . -type f -iname '*.cpp' -exec chmod a-x '{}' ';'
+%description devel
+Header files and scripts for mstflint
+
+%prep
+%setup -q
 
 %build
-./autogen.sh
-%configure --enable-fw-mgr --enable-adb-generic-tools
+
+MSTFLINT_VERSION_STR="%{name} %{version}-%{release}"
+%configure --enable-fw-mgr --enable-adb-generic-tools MSTFLINT_VERSION_STR="${MSTFLINT_VERSION_STR}"
 %make_build
 
 %install
-%make_install
-# Remove the devel files that we don't ship
-rm -fr %{buildroot}%{_includedir}
-find %{buildroot} -type f -name "*.la" -delete -print
-find %{buildroot} -type f -name '*.a' -delete
```

Comment:
I don't think we want to add static libraries in general, unless we have a need for them can we remove them?